### PR TITLE
[Android] update native example to run NNFW

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update native example, including nnfw sample code.
Also add SNAP-tensorflow and remove jave code to initialize nnstreamer.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c [hunk 1]
```diff
@@ -170,6 +171,186 @@ run_snap_example (void)
...
+             "tensor_filter framework=nnfw model=/sdcard/nnstreamer/tflite_model_add/add.tflite ! "
+             "tensor_sink name=sinkx";
+
+  LOGI ("Start to run NNStreamer pipeline with NNFW.");
+
+  /* construct a pipeline */
+  status = ml_pipeline_construct (description, NULL, NULL, &pipe);
+  if (status != ML_ERROR_NONE) {
+    LOGE ("Failed to construct the pipeline.");
+    goto done;
```
Comment (dongju-chae) [inline android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c:200]: Is it okay to destroy the pipeline even when `ml_pipeline_construct()` is failed` ?

File: android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c [hunk 2]
```diff
@@ -170,6 +171,186 @@ run_snap_example (void)
...
+  int status = ML_ERROR_NONE;
+  float *data = NULL;
+  size_t data_size;
+
+  const char nnfw_model[] = "/sdcard/nnstreamer/tflite_model_add/add.tflite";
+
+  status = ml_single_open (&single, nnfw_model, NULL, NULL, ML_NNFW_TYPE_NNFW, ML_NNFW_HW_CPU);
+  if (status != ML_ERROR_NONE) {
+    LOGE ("Failed to push input data.");
+    goto done;
```
Comment (dongju-chae) [inline android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c:290]: same issue.

File: android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c (hunk 1 above)
Comment (jaeyun-jung) [reply android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c:200]: No crash issue. If failed to construct pipeline, pipe is null. Then destroy() will return invalid-param error when given handle is null.

Comment (dongju-chae) [reply android/example_app/capi-sample/src/main/jni/nnstreamer-native-sample.c:200]: ok

**Comment by jaeyun-jung** [comment]
2020-04-17 PR updated
- add native init (https://github.com/nnstreamer/nnstreamer/pull/2280)
- add snap-tensorflow pipeline example

**Comment by jaeyun-jung** [comment]
2020-04-20 PR updated
- do not run the examples on UI thread

File: android/example_app/capi-sample/src/main/java/org/nnsuite/nnstreamer/sample/MainActivity.java [hunk 3]
```diff
@@ -32,15 +30,6 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
```
Comment (myungjoo) [inline android/example_app/capi-sample/src/main/java/org/nnsuite/nnstreamer/sample/MainActivity.java:31]: We may need to start thinking about nnstreamer version-checking for later similar changes that differ per nnstreamer version.

**Pull request merged**